portfolio/src/main/webapp/style.css

```diff
@@ -2,49 +2,131 @@
 @include button.core-styles;
 @use "@material/card";
 @include card.core-styles;
+@import url('https://fonts.googleapis.com/css?family=Lato:100,100i,300,300i,400,400i,700,700i,900,900i');
 
+html {
+   background-size: cover;
+   background-color: #6495ED;
+   height:100%;
+}
+
+/*
+ * Credit to codepen: https://codepen.io/valentin/pen/fkdCG
+*/
 body {
-  background-color: #8EC5FC;
-  background-image: linear-gradient(62deg, #8EC5FC 0%, #E0C3FC 100%);
+  animation: wind 20s linear infinite;
+  background-color: #808080;
+  background: url(https://static.radulescu.me/examples/clouds/clouds1000.png);
+  background: url(https://static.radulescu.me/examples/clouds/clouds1000.png) 0 200px,
```

Comment:
can these two background property be consolidated into one?

Reply:
Yes indeed! thanks & updated!